Makefile

```diff
@@ -15,3 +15,17 @@ export Q
 # ---------------------------------------------------------------------------
 htmldocs:
 	$(Q)$(MAKE) -C doc htmldocs
+
+# SHELL used by kbuild
```

Comment:
Kbuild is the name for the Makefiles in the Linux kernel, so this comment might look confusing.